Marlin/src/feature/power_loss_recovery.cpp

```diff
@@ -392,7 +392,7 @@ void PrintJobRecovery::resume() {
 
   // Resume the SD file from the last position
   char *fn = info.sd_filename;
-  while (*fn == '/') fn++;
+  // while (*fn == '/') fn++;
```

Comment:
Can simply delete this line instead of commenting it out. Git retains history of what was there. Id also suggest a rebase to fix the menu issue causing this to fail travis.